sysID/fit_flywheel_philip.py

```diff
@@ -0,0 +1,180 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from models import cartpole_flywheel as model
+
+# Load the data
+q = np.load('/Users/philipqueen/Documents/Humon Research Lab/FreeMocap_Data/4stepsequence_session2_10_5_22/pendulum_flywheel_state_q.npy')
+qDot = np.load('/Users/philipqueen/Documents/Humon Research Lab/FreeMocap_Data/4stepsequence_session2_10_5_22/pendulum_flywheel_state_qdot.npy')
+qDotDot = np.load('/Users/philipqueen/Documents/Humon Research Lab/FreeMocap_Data/4stepsequence_session2_10_5_22/pendulum_flywheel_state_qddot.npy')
+
+params = model.default_params()
+
+# convert to meters
+q[:, 0] = q[:, 0]/1000.
+
+qDot[:, 0] = qDot[:, 0]/1000.
+
+qDotDot[:, 0] = qDotDot[:, 0]/1000.
+
+# convert angle to radians
+q[:, 1] = q[:, 1]*np.pi/180.
+qDot[:, 1] = qDot[:, 1]*np.pi/180.
+qDotDot[:, 1] = qDotDot[:, 1]*np.pi/180.
+
+# convert to meters squared
+q[:, 2] = q[:, 2]/1000000.
```

Comment:
same as with arm+flywheel, I don't understand why the flywheel angle should be in meters squared? I think this should be in radians, like the pendulum angle. How did you fit the data?